includes/functions_get.php

```diff
@@ -323,6 +323,8 @@ function tsml_get_meeting($meeting_id = false)
         }
         if (in_array($key, $tsml_array_fields, true)) {
             $value = array_values((array) maybe_unserialize($value));
+        } elseif (in_array($key, $tsml_url_fields, true)) {
```

Comment:
one note here, the `data_source` url was getting it's entities converted, so `https://site.com/?feed&prop=2` was becoming `https://site.com?feed&amp;prop=2` - ampersand convert. Added logic to avoid this for url fields.